sdk/core/AzureCore/Source/Pipeline/Policies/ContentDecodePolicy.swift

```diff
@@ -169,15 +171,15 @@ public class ContentDecodePolicy: PipelineStage {
     // MARK: Properties
 
     public var next: PipelineStage?
-    public let jsonRegex = NSRegularExpression("^(application|text)/([0-9a-z+.]+)?json$")
     public var logger: ClientLogger?
 
-    internal var delegate: ContentDecodeXMLParser?
+    internal let jsonRegex = NSRegularExpression("^(application|text)/([0-9a-z+.]+)?json$")
+    internal var xmlParser: ContentDecodeXMLParser?
```

Comment:
Swiftlint complained that, as a delegate, this should be a weak variable. However, for this to be a true delegate pattern, ContentDecodeXMLParser should be a protocol, not a class. Realizing that a protocol was not only unwanted but unnecessary, I changed this to not be called a delegate, as there's no opportunity here for a memory leak.